apps/src/templates/rubrics/RubricFloatingActionButton.jsx

```diff
@@ -55,6 +57,8 @@ export default function RubricFloatingActionButton({
     }
   }, [eventData, studentLevelInfo]); // Neither of these should change, so this should run once
 
+  const icon = aiEnabled ? aiFabIcon : rubricFabIcon;
```

Comment:
looks good for this PR, but wondering @tess323 if we also want to show the rubric-only icon on lesson 7, where we will have a rubric but no ai yet? unsure if this will be a common case for us